r/vignettes/install.Rmd

```diff
@@ -517,10 +488,6 @@ The install script should work everywhere, so if libarrow fails to compile,
 please [report an issue](https://issues.apache.org/jira/projects/ARROW/issues)
 so that we can improve the script.
 
-### Known installation issues
```

Comment:
Similarly, maybe this should stick around in some form since we direct people here for troubleshooting.